### PR TITLE
IGVF-1143-titles

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/biosample.json [hunk 1]
```diff
@@ -126,21 +126,21 @@
...
-            "description": "The biosample(s) this biosample is pooled from.",
+            "title": "Biosamples Pooled From",
+            "description": "The biosamples this biosample is pooled from.",
             "type": "array",
             "uniqueItems": true,
             "minItems": 2,
             "items": {
                 "title": "Biosample",
-                "description": "Biosample(s).",
+                "description": "A biosample this biosample is pooled from.",
```
Comment (gabdank) [inline src/igvfd/schemas/biosample.json:143]: Maybe "Pooled biosample." would be more clear?

File: src/igvfd/schemas/construct_library.json [hunk 2]
```diff
@@ -279,6 +282,7 @@
                     "minItems": 1,
                     "uniqueItems": true,
                     "items": {
+                        "title": "Clone Gene",
```
Comment (gabdank) [inline src/igvfd/schemas/construct_library.json:285]: should be "Cloned gene"

File: src/igvfd/schemas/in_vitro_system.json [hunk 3]
```diff
@@ -176,6 +176,7 @@
         },
         "pooled_from": {
             "items": {
+                "title": "Biosample Pooled From",
```
Comment (gabdank) [inline src/igvfd/schemas/in_vitro_system.json:179]: would not the title come from the Biosample?

**Review by gabdank — CHANGES_REQUESTED** [review]
few small suggestions

File: src/igvfd/schemas/in_vitro_system.json (hunk 3 above)
Comment (ian-whaling) [reply src/igvfd/schemas/in_vitro_system.json]: You're right, changed.

File: src/igvfd/schemas/construct_library.json (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/schemas/construct_library.json]: Changed.

File: src/igvfd/schemas/biosample.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/biosample.json]: changed.

**Review by gabdank — CHANGES_REQUESTED** [review]
please resole conflict src/igvfd/schemas/construct_library.json

**Review by gabdank — APPROVED** [review]

**Pull request merged**